### PR TITLE
Use globals instead of imports in tutorials

### PR DESCRIPTION
**Pull request by oandregal**
For reference https://github.com/WordPress/gutenberg/pull/13954

### CODE REVIEW

**Review by gziolo — APPROVED** [review]
Yes, it's fine based on the comments from the referenced issue.

By the way, can you also update import statement in another file:
https://github.com/WordPress/gutenberg/blob/master/docs/designers-developers/designers/menu-item.md#usage-1

File: docs/designers-developers/developers/tutorials/metabox/meta-block-3-add.md [hunk 1]
```diff
@@ -60,8 +60,9 @@ Add this code to your JavaScript file (this tutorial will call the file `myguten
 ```
 {% ESNext %}
 ```jsx
-import { registerBlockType } from '@wordpress/blocks';
-import { TextControl } from '@wordpress/components';
+
+var registerBlockType = wp.blocks.registerBlockType;
```
Comment (gziolo) [inline docs/designers-developers/developers/tutorials/metabox/meta-block-3-add.md]: By the way, it's ESNext, so I would suggest updating to:
```suggestion
const { registerBlockType } = wp.blocks;
```

File: docs/designers-developers/developers/tutorials/metabox/meta-block-3-add.md [hunk 2]
```diff
@@ -60,8 +60,9 @@ Add this code to your JavaScript file (this tutorial will call the file `myguten
...
+var TextControl = wp.components.TextControl;
```
Comment (gziolo) [inline docs/designers-developers/developers/tutorials/metabox/meta-block-3-add.md]:
```suggestion
const { TextControl } = wp.components;
```

**Review by gziolo — CHANGES_REQUESTED** [review]

**Comment by nosolosw** [comment]
> By the way, can you also update import statement in another file:

Yeah, I've also found another use of `import` in the documents. https://github.com/WordPress/gutenberg/blob/master/docs/designers-developers/developers/feature-flags.md

I'm on the fence about those two. If I lean towards not modifying them is because they look like docs for people developing Gutenberg not 3rd party plugins/themes :man_shrugging:

**Review by gziolo — APPROVED** [review]
:shipit: for the file included in this PR.

**Comment by nosolosw** [comment]
An unrelated e2e test is failing:

```bash
FAIL packages/e2e-tests/specs/blocks/preformatted.test.js
  - Preformatted › should preserve character newlines
    No node found for selector: button[aria-label="More options"]
      at assert (../../node_modules/puppeteer/lib/helper.js:282:11)
      at Frame.click (../../node_modules/puppeteer/lib/FrameManager.js:590:5)
```

**Comment by gziolo** [comment]
It looks like a regression. I'm seeing it in many other PRs.

**Pull request merged**